scripts/nasa/cmip6_sea_level/test_data/expected/total_ssp119_medium_confidence_rates.csv

```diff
@@ -13,6 +13,20 @@ dcid:Earth,dcid:Min_RateOfChange_SeaLevel_SSP119,2120,0.0,dcs:MillimeterPerYear,
 dcid:Earth,dcid:Min_RateOfChange_SeaLevel_SSP119,2130,-0.2,dcs:MillimeterPerYear,dcs:IPCC_MediumConfidence
 dcid:Earth,dcid:Min_RateOfChange_SeaLevel_SSP119,2140,-0.3,dcs:MillimeterPerYear,dcs:IPCC_MediumConfidence
 dcid:Earth,dcid:Min_RateOfChange_SeaLevel_SSP119,2150,-0.4,dcs:MillimeterPerYear,dcs:IPCC_MediumConfidence
+dcid:Earth,dcid:Percentile10_RateOfChange_SeaLevel_SSP119,2020,3.0,dcs:MillimeterPerYear,dcs:IPCC_MediumConfidence
```

Comment:
@pradh Any idea why these lines are being added for a mere version change?

Reply:
Perhaps your `round(quantile, 1)` is actually important even otherwise?